### PR TITLE
[PREVIEW] Is life intolerable

### PR DESCRIPTION
**Pull request by mounikahmcts**
# Description

Changes for Adultery Q: Is life intolerable?

Fixes # DIV-3159

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Dependent changes  -  https://github.com/hmcts/div-decree-nisi-frontend/pull/63  - This needs to be merged to test these changes completely

### CODE REVIEW

File: test/unit/steps/intolerable.test.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  it('redirects to exitIntolerable if answered no', () => {
+    const fields = {
+      intolerable: 'no'
+    };
+    return question.redirectWithField(Intolerable, fields, ExitIntolerable);
+  });
+
+  it('returns correct answers if answered yes', () => {
+    const expectedContent = [
+      // eslint-disable-next-line max-len
```
Comment (tchow8) [inline test/unit/steps/intolerable.test.js]: Might not need the eslint-disable? Doesn't seem this line is that long?
Same below.

File: steps/exit-intolerable/ExitIntolerable.step.js [hunk 2]
```diff
@@ -9,6 +10,10 @@ class ExitIntolerable extends Page {
   get session() {
     return this.req.session;
   }
+
+  get flowControl() {
```
[the remark this answers is not in the document]
Comment (mounikahmcts) [reply steps/exit-intolerable/ExitIntolerable.step.js]: > whats this for?

Added this flow control to stop the flow here. flowcontrol is not defined in page.js without which the next page cannot be decided

[the remark this answers is not in the document]
Comment (dtaylor7) [reply steps/exit-intolerable/ExitIntolerable.step.js]: 👍

**Comment by nsingamsetty** [comment]
This is tested and working fine and going to merge this PR.

There is a back button issue which is already reported in DIV-3336 and the related scenario which is failing here is mentioned there. Hence this is merging and closing the ticket.

**Pull request merged**